ruby/rails6-postgres/vector/vector.toml.template

```diff
@@ -0,0 +1,27 @@
+[sources.internal_logs]
+type = "internal_logs"
+
+[sources.postgres]
+type = "postgresql_metrics"
+endpoints = [ "postgresql://$POSTGRES_USER:$POSTGRES_PASSWORD@postgres:5432/$POSTGRES_DB" ]
+
+[transforms.modify]
+type = "remap"
+inputs = ["postgres"]
+source = '''
+if starts_with!(.name, "pg_") {
+  .name = join!(["vector_", .name])
+} else {
+  .name = join!(["vector_pg_", .name])
+}
+'''
```

Comment:
See discussion in https://github.com/appsignal/public_config/pull/54#issuecomment-1709962675 about if we should keep this transformation or not.

Reply:
Removing this transform.